v2/pkg/cli/prepare.go

```diff
@@ -0,0 +1,211 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	_ "github.com/distribution/distribution/v3/registry/storage/driver/filesystem"
+	"github.com/google/uuid"
+	"github.com/openshift/oc-mirror/v2/pkg/additional"
+	"github.com/openshift/oc-mirror/v2/pkg/config"
+	clog "github.com/openshift/oc-mirror/v2/pkg/log"
+	"github.com/openshift/oc-mirror/v2/pkg/manifest"
+	"github.com/openshift/oc-mirror/v2/pkg/mirror"
+	"github.com/openshift/oc-mirror/v2/pkg/operator"
+	"github.com/openshift/oc-mirror/v2/pkg/release"
+	"github.com/spf13/cobra"
+)
+
+// NewPrepareCommand - setup all the relevant support structs
+// to eventually execute the 'prepare' sub command
+func NewPrepareCommand(log clog.PluggableLoggerInterface) *cobra.Command {
```

Comment:
It is always better to have code separated by files. Thanks for moving it to a different file. 

I'm wondering if we always need to duplicate the code for each subcommand, am I missing anything maybe?